### PR TITLE
Include number of failed stories in Chromatic comment

### PR DESCRIPTION
**Pull request by HalvorHaugan**
[Docs](https://www.chromatic.com/docs/github-actions/#outputs)

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -109,5 +109,8 @@ jobs:
 
             ### [Endringer til review](${{steps.chromatic_tests.outputs.url}}): ${{steps.chromatic_tests.outputs.changeCount}}
 
+            ${{ steps.chromatic_tests.outputs.errorCount > 0 && format('### ❌ Stories med feil: {0}', steps.chromatic_tests.outputs.errorCount) || '' }}
+            ${{ steps.chromatic_tests.outputs.interactionTestFailuresCount > 0 && format('### ❌ Feilede interaksjonstester: {0}', steps.chromatic_tests.outputs.interactionTestFailuresCount) || '' }}
```
Comment (HalvorHaugan) [inline .github/workflows/chromatic.yml:104]: Haven't been able to figure out what will cause `interactionTestFailuresCount` to be > 0 🤔

Let me know if you know of a cleaner way to do conditionals or string concatenations 😄

Comment (KenAJoh) [reply .github/workflows/chromatic.yml:104]: Will it not be be these tests that causes `interactionTestFailuresCount` to work?

Screenshot 2024-10-31 at 12 11 15

Comment (HalvorHaugan) [reply .github/workflows/chromatic.yml:104]: You would think that, but everything I tried only affected `errorCount`. Look at the commits to see what I tried.

**Review by KenAJoh — DISMISSED** [review]
👏 👏 👏 👏 👏 👏 👏 
Since we have to go intro chromatic either way when there are changes or errors detected, would it make sense to have it as a link at the top with "storybook demo"? So something like:

```
Storybook demo <- link to storybook
Chromatic <- link to chromatic

---

Endringer til review <- not a link anymore
Feil... <- not a link
```

**Review by KenAJoh — APPROVED** [review]

**Pull request merged**